lib/pillar/type_convert/to_elixir.ex

```diff
@@ -102,4 +102,8 @@ defmodule Pillar.TypeConvert.ToElixir do
   def convert("Decimal" <> _decimal_subtypes, value) do
     value
   end
+
+  def convert("Map" <> _subtypes, value) do
+    value
```

Comment:
Does recursive type convert work here?

I'm sure at least we can try get subtypes and convert them. `Map(UInt8, String)`